UI/src/view/point_asset/massTransitStopForm.js

```diff
@@ -985,12 +1098,86 @@
               'Olet siirtämässä pysäkin ELYn ylläpitoon! Huomioithan, että osa pysäkin varustetiedoista saattaa kadota tallennuksen yhteydessä.',
               {type: 'alert'});
         }
+
+        if (property.publicId === 'palvelu') {
+          hideOrShowTarkenne();
+          updateViranomaisdataaValue();
+
+          pointAssetToSave = true;
+        }
+
       });
 
       eventbus.on('terminalBusStop:selected', function(value) {
-        isTerminalBusStop = value;
+        busStopTypeSelected = value;
```

Comment:
Don't remove the way to validate what kind of Bus Stop is selected and then strat spreading validations like:
busStopTypeSelected == 6 OR busStopTypeSelected == 7.

Create a new method to validate this, because in the future when we want modify the type value will be a mess

Reply:
@marquesfm Check modification on this code

Reply:
Verified